includes/localization.php

```diff
@@ -1,28 +1,29 @@
 <?php
-function pmpro_load_textdomain()
-{
-    //get the locale
-	$locale = apply_filters("plugin_locale", get_user_locale(), "paid-memberships-pro");
-	$mofile = "paid-memberships-pro-" . $locale . ".mo";
+function pmpro_load_textdomain() {
+	$locale = apply_filters( 'plugin_locale', get_user_locale(), 'paid-memberships-pro' );
+	$mofile = esc_attr( 'paid-memberships-pro-' . $locale . '.mo' );
 
 	//paths to local (plugin) and global (WP) language files
-	$mofile_local  = dirname(__FILE__)."/../languages/" . $mofile;
+	$mofile_local  = dirname( __DIR__ ) . '/languages/' . $mofile;
 	$mofile_global = WP_LANG_DIR . '/pmpro/' . $mofile;
 	$mofile_global2 = WP_LANG_DIR . '/paid-memberships-pro/' . $mofile;
 
+	unload_textdomain( 'paid-memberships-pro' );
```

Comment:
You have to unload the textdomain as it autoloads. Other popular plugins do it this way as well, as we're loading the textdomain directly after that.